windows/check_quiesce_snapshot/check_quiescing_type.yml

```diff
@@ -89,6 +89,8 @@
       when:
         - guestinfo_guest_full_name | regex_search('Windows Server ([0-9]{4})')
         - guestinfo_guest_full_name | regex_search('Windows Server ([0-9]{4})') == guest_os_ansible_distribution | regex_search('Windows Server ([0-9]{4})')
+        - win_boot_disk_ctl_type != 'ide'
```

Comment:
The two conditions can be merged into one.
win_boot_disk_ctl_type not in ['ide','sata']

Reply:
Right, thanks.